README.md

```diff
@@ -48,6 +48,7 @@ ng add @auth0/auth0-angular
 - [Display the user profile](#display-the-user-profile)
 - [Protect a route](#protect-a-route)
 - [Call an API](#call-an-api)
+- [Specifying Configuration using APP_INITIALIZER](#specifying-configuration-using-app_initializer)
```

Comment:
Package consumers may be unaware of the "APP_INITIALIZER" implementation detail.

Perhaps a descriptor including intent such as "Runtime / Async / Provider / APP_INITIALIZER Configuration" could broaden the documentation reach.

Reply:
I like runtime or dynamic configuration, we could also use dynamic configuration. APP INITIALIZER is an implementation detail, even though it is currently the only supported way to use dynamic configuration.